### PR TITLE
Nvyn

### PR DESCRIPTION
**Pull request by NazarVynnyk**
Fix code style

### CODE REVIEW

**Comment by NazarVynnyk** [comment]
merge to develop

File: RESTserver/src/main/java/com/softserve/if072/restservice/controller/ExceptionHandlerController.java [hunk 1]
```diff
@@ -39,4 +39,12 @@ public String accessDeniedException(AccessDeniedException e) {
         LOGGER.error(e.getMessage());
         return e.getMessage();
     }
+
+    @ExceptionHandler(IllegalArgumentException.class)
+    @ResponseStatus(HttpStatus.EXPECTATION_FAILED)
+    public String IllegaArgumentException(IllegalArgumentException e) {
+        LOGGER.error(e.getMessage());
```
Comment (kucherif) [inline RESTserver/src/main/java/com/softserve/if072/restservice/controller/ExceptionHandlerController.java]: `LOGGER.error(e.getMessage(), e);`

File: RESTserver/src/main/java/com/softserve/if072/restservice/service/ShoppingListService.java [hunk 2]
```diff
@@ -35,7 +36,7 @@ public ShoppingListService(ShoppingListDAO shoppingListDAO) {
 
     public ShoppingList getByUserAndProductId(int user_id, int product_id) throws DataNotFoundException {
         ShoppingList list = shoppingListDAO.getByUserAndProductId(user_id, product_id);
-        if (list != null) {
+        if (!CollectionUtils.isEmpty((Collection<?>) list)) {
```
Comment (kucherif) [inline RESTserver/src/main/java/com/softserve/if072/restservice/service/ShoppingListService.java]: khm, what is this? Shopping list is a plain object, why do you cast it to the list?

File: RESTserver/src/main/java/com/softserve/if072/restservice/service/StoreService.java [hunk 3]
```diff
@@ -77,10 +77,14 @@ public Store getStoreByID(int id) throws DataNotFoundException {
...
+     * @throws IllegalArgumentException - if the passed store is null or has empty name
+     * not found
      */
     @Transactional
-    public void addStore(Store store) {
-        storeDAO.insert(store);
+    public void addStore(Store store) throws IllegalArgumentException {
+        if (store != null && store.getName() != null && store.getName() != "") {
+            storeDAO.insert(store);
+        } else throw new IllegalArgumentException("Illegal arguments!");
```
Comment (kucherif) [inline RESTserver/src/main/java/com/softserve/if072/restservice/service/StoreService.java]: please format code. Add argument to the exception message

File: RESTserver/src/main/java/com/softserve/if072/restservice/service/ShoppingListService.java (hunk 2 above)
Comment (NazarVynnyk) [inline RESTserver/src/main/java/com/softserve/if072/restservice/service/ShoppingListService.java]: That was my mistake

**Pull request merged**